src/test/crimson/seastore/transaction_manager_test_state.h

```diff
@@ -395,7 +395,7 @@ DeviceRef EphemeralSegmentedDevices::get_primary_device_ref() {
   return std::make_unique<TestSegmentManagerWrapper>(*segment_manager);
 }
 
-class TestRandomBlockDeviceWrapper final : public Device {
```

Comment:
I'm trying to understand why do we need `TestSegmentManagerWrapper` and `TestRandomBlockDeviceWrapper` initially. Seems to me the only requirement is to skip `Device::mount()` and `Device::mkfs()` because they are handled by UT instead.

If true, it'll be better to drop both wrappers and expose `test_mkfs()` and `test_mount()` from `SeaStore`. The common logics can be encapsulated and be shared with the existing `SeaStore::mkfs()` and `mount()`.

Reply:
Sound reasonable to me.